tools/project-dashboard/src/client/styles.css

```
@@ -0,0 +1,200 @@
+:root {
+  font-family: "SFMono-Regular", "Cascadia Code", "Roboto Mono", monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)